client/src/views/TrackViewer/Layers.vue

```diff
@@ -205,7 +205,7 @@ export default defineComponent({
     editAnnotationLayer.$on('update:geojson',
       (data: GeoJSON.Feature<GeoJSON.Polygon>) => {
         const bounds = geojsonToBound(data);
-        emit('update-rect-bounds', frameNumber.value, bounds);
+        emit('update-rect-bounds', frameNumber.value, bounds, filename.value);
```

Comment:
If we truly have the filename added on export time, we probably don't need to have the filename in the layers.  So we can remove this here and make sure that the `update-rect-bounds`, doesn't depend on having a filename as well.